pdr_backend/lake/etl.py

```diff
@@ -153,7 +152,7 @@ def do_bronze_step(self):
 
     def _get_max_timestamp_values_from(
         self, table_names: List[str], table_type: TableType = TableType.NORMAL
```

Comment:
I don't really agree with having the same table type for a list of table_names, it looks like it could cause troubles in the future and is not flexible for mixed queries.

Reply:
my best suggestion would be a Table (or NamedTable...?) class which would hold the table name and type. It would support direct access to the attributes and make for better separation of concerns.

i.e. tables[List[NamedTable]] instead of table_names, get rid of table_type param altogether
then "if table.name not in all_db_tables"...

The NamedTable class would of course hold functionality to get_table_with_name and the initialisation would happen outside this function.

Such a change would greatly improve other parts of the code as well, where such concerns are mixed and prone to bugs.